assets/js/components/Header.js

```diff
@@ -31,7 +36,7 @@ import ErrorNotification from '../components/notifications/error-notification';
 import { STORE_NAME as CORE_USER } from '../googlesitekit/datastore/user/constants';
 const { useSelect } = Data;
 
-const Header = () => {
+const Header = ( { children } ) => {
```

Comment:
The IB/issue doesn't mention any VRT changes expected, but I see this in all the headers:

<img width="912" alt="Screenshot 2020-11-18 at 13 40 17" src="https://user-images.githubusercontent.com/90871/99537527-b51e4500-29a3-11eb-8204-e6c4a0890822.png">

The dropdown indicator is gone. I can't seem to access the Figma file to see the mocks—is that intentional? It seems odd to remove the indication that it's clickable, but then it's similar to the style on google.com so I guess it's fine. Just making sure.

Reply:
I'll send you a screenshot in Slack, the new designs remove this arrow and increase the size of the artwork to 36px square.